bazel/repositories.bzl

```diff
@@ -596,7 +596,14 @@ def _com_googlesource_quiche():
     )
 
 def _com_github_grpc_grpc():
-    _repository_impl("com_github_grpc_grpc")
+    _repository_impl(
+        "com_github_grpc_grpc",
+        patches = [
+            "@envoy//bazel:grpc-protoinfo-1.patch",
```

Comment:
Can this two patch combined to one?

Reply:
I certainly can, but are you sure that's what you want? This way, it each `.patch` file corresponds to a gRPC commit, which I thought was a nice property.

Reply:
Ah I see, sure let’s keep as it is.